docs/src/modules/utils/helpers.ts

```diff
@@ -226,13 +226,18 @@ export function getDependencies(
  * @return The cookie value
  */
 export function getCookie(name: string): string | undefined {
-  if (typeof document !== 'undefined') {
-    const value = `; ${document.cookie}`;
-    const parts = value.split(`; ${name}=`);
-    if (parts.length === 2) {
-      return parts[1].split(';').shift();
-    }
+  if (typeof document === 'undefined') {
```

Comment:
Adding a test would be nice.

Reply:
What would the test be about?

Reply:
getCookie function that it returns the correct result. No strong opinion, it is just easier to me if there is a test I can see for input & output.

Reply:
Oh I see, I'm not aware of a precedant to tests cookies, it could be interesting. I'm in the same boat, I don't really know how this function works.